README.md

```diff
@@ -202,6 +202,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [Apify](https://www.apify.com/) — Web scraping and automation platform to create an API for any website and extract data. Ready-made scrapers, integrated proxies, and custom solutions. Free plan with $5 platform credits included every month.
   * [API Mocha](https://apimocha.com) - Completely free online API mocking for testing and prototyping.  Make up to 500 requests per day, fully customizable API responses, download mock rules as a Postman collection.
   * [APITemplate.io](https://apitemplate.io) - Auto-generate images and PDF documents with a simple API or automation tools like Zapier & Airtable. No CSS/HTML required. Free plan comes with 50 images/month and 3 templates.
+  * [Arize AI](https://arize.com/) - Arize AI free observability tool for monitoring machine learning models in production and root-causing issues such as data quality and performance drift.
```

Comment:
We list entries only once, please pick the best place for it.  Also add something like `free for 2 models`.

Reply:
Will do!